### PR TITLE
Document the GitHub Apps used in the Jenkins Infra

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Documents GitHub Apps which we have created to address different use-cases in our infra

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
CC @timja @olblak @MarkEWaite @daniel-beck . The GitHub apps documentation is ready for review

File: bots.adoc [hunk 1]
```diff
@@ -54,11 +54,15 @@ ex:
 |===
 
 
-This bots is the first github bot used in the Jenkins Infrastructure
+This bots is the first github bot used in the Jenkins Infrastructure.
```
Comment (timja) [inline bots.adoc:57]:
```suggestion
This bot is the first GitHub bot used in the Jenkins Infrastructure.
```

File: bots.adoc [hunk 2]
```diff
@@ -54,11 +54,15 @@ ex:
...
 ex:
 
 * Automatically open pull request
 * jenkinsadmin irc bot
+* Some Pipeline steps and operations on ci.jenkins.io
 
 Remark: We don't have access to this bot anymore and we need Kohsuke to transfer the ownership
+
+Remark: Before July 30th, 2020 the bot was also used in GitHub org Folder organizations on ci.jenkins.io.
+At the moment it is replaced by link:/github-apps.adoc#cijenkinsio[ci.jenkins.io GitHub App].
```
Comment (timja) [inline bots.adoc:68]:
```suggestion
It is replaced by link:/github-apps.adoc#cijenkinsio[ci.jenkins.io GitHub App].
```

**Comment by oleg-nenashev** [comment]
I have no merge permissions in this repo. Would appreciate if someone
merges the docs

**Comment by MarkEWaite** [comment]
No merge permission for me either on this repo

**Pull request merged**